third_party/move/move-model/bytecode/tests/reaching_def/basic_test.exp

```diff
@@ -57,8 +57,8 @@ fun ReachingDefTest::basic($t0|a: u64, $t1|b: u64): u64 {
 
 
 [variant baseline]
-fun ReachingDefTest::create_resource($t0|sender: signer) {
-     var $t1: signer
+fun ReachingDefTest::create_resource($t0|sender: &signer) {
+     var $t1: &signer
```

Comment:
how do changes like this make it in here?

these are generated, right?

curious if this will also change the resulting byte code?

Reply:
As mentioned in the PR description, for some of those tests, previously we run some bytecode transformations specific to the prover before the tested feature's transformation (here, reaching_defs is not longer preceded by reference elimination). This is not longer possible after splitting the crates.